### PR TITLE
Added a few data entries.

### PR DESCRIPTION
**Pull request by Scrivener07**

### CODE REVIEW

**Comment by Ortham** [comment]
Thanks - could you add `skyrim` as a keyword to these though? `Unforbidable Patcher` mentions the game in its description, but the others don't, and it can be useful to see that info.

**Comment by Scrivener07** [comment]
Added the keywords to the entries.

File: data.json [hunk 1]
```diff
@@ -1019,5 +1019,38 @@
...
+    "name": "MCM Designer",
+    "keywords": [
+      "skyrim",
+      "papyrus",
+      "scripting"
+    ],
+    "description": "An application that generates papyrus scripts for MCM menus.",
+    "homepage": "http://www.nexusmods.com/skyrim/mods/29821/",
+    "repository": "https://github.com/Daimakaicho/MenuDesigner"
+  },
```
Comment (Ortham) [inline data.json]: Hmm, I'm not sure why the Travis status check isn't working, but I validated it manually, and this trailing comma is invalid. Could you remove it?

**Comment by Scrivener07** [comment]
I removed that sneaky trailing comma.

**Comment by Ortham** [comment]
Whoops, wrong button...

Looks like Travis is having some problems, I'll just disable the status check.

**Pull request merged**